astropy/timeseries/periodograms/lombscargle/core.py

```diff
@@ -502,14 +502,16 @@ def design_matrix(self, frequency, t=None):
         ----------
         frequency : float
             the frequency for the model
-        t : array_like or `~astropy.units.Quantity` or `~astropy.time.Time`, length n_samples
-            times at which to compute the model (optional). If not specified,
-            then the times and uncertainties of the input data are used
+        t : array_like or `~astropy.units.Quantity` or `~astropy.time.Time`
+            length ``n_samples`` times at which to compute the model
+            (optional). If not specified, then the times and uncertainties of
+            the input data are used
 
         Returns
         -------
-        X : np.ndarray (len(t), n_parameters)
+        X : np.ndarray
             The design matrix for the model at the given frequency.
+            Shape (len(t), n_parameters)
```

Comment:
This should have a shape of ``(len(t), n_parameters)``

Reply:
✅